source/refunding_payments/index.html.md.erb

```diff
@@ -287,8 +287,11 @@ You can use the following query parameters to filter refunds by date:
 * `from_date`, which is the start date to search, inclusive
 * `to_date`, which is the end date to search, exclusive
 
+All dates are in Coordinated Universal Time (UTC).
+
 Dates must be in [ISO
-8601 format](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard). For example `2019-09-20T13:30:00Z`.
+8601 format](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard).
```

Comment:
Just an idea, but I wonder whether bullet format might make this a bit clearer. I think the descriptions could be made a little more concise too (for example "a time value precise to at least hours and minutes" could be shortened to "include hours and minutes")

For example:

Dates must be in [ISO 8601 format] and include a:

- date
- hours and minutes - seconds are optional
- time zone

For example `2019-09-20T13:30Z`.